### ISSUE TITLE
The term "relay" appears in the topology diagram and introduction but is not described in detail in the rest of the specification

### ISSUE THREAD

**Issue by mwherman2000** [decentralized-identity/decentralized-web-node#151]
The term _relay_ appears in the topology diagram (https://identity.foundation/decentralized-web-node/spec/#topology) and introduction but is not described in detail in the rest of the specification.  I  believe _relay_ is a key concept and should be detailed later on in the specification.

**Comment by andorsk** [comment]
Agreed we can further clarify the word "relay" in the spec. Thanks @mwherman2000 !
